src/Pages/Dashboard/DashboardLayout.jsx

```diff
@@ -0,0 +1,36 @@
+import react, {useEffect} from 'react';
+import { useLayout, useAlert } from '../../Context';
+import Dashboard_route from '../../route/Dashboard_route';
+import { Alert } from '../../Shared';
```

Comment:
club the imports alert and header etc seem to be coming from the same file

Reply:
done!